.gitlab-ci.yml

```diff
@@ -0,0 +1,28 @@
+stages:
+  - validation
+  - build
+
+json:
```

Comment:
This is not needed here.

Reply:
I meant the whole file.

Reply:
Fixed with https://github.com/flathub/flathub/pull/771/commits/9cb8de9106a80a05d36925aa441b877e839b4fd4.